docs/criufeature.md

```diff
@@ -0,0 +1,93 @@
+<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# CRIU feature
+
+**(Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only)**
+
+:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** CRIU support is a technical preview feature that is intended for container environments. All APIs and options are subject to change. CRIU support requires the version of CRIU that is a package in Semeru UBI container images.
```

Comment:
Why is the mention of container images in a Note? Doesn't seem like something to call out specially - I would incorporate it into the rest of the text instead. Or is it there because users shouldn't use CRIU in non-container environments for some reason?
This is the first instance of CRIU in normal text so it should be expanded.
I think we should mention something about using this preview in production, since that's normally not supported for tech previews but it is in this case. Eg "Support for the Checkpoint/Restore in User Space (CRIU) tool is currently provided as a technical preview. This preview is supported for use in production environments, however, all APIs and options are subject to change."
The last sentence seems like internal info that should be removed. Or if it is trying to convey info to the user, it should be clarified. For example, are we trying to say that the CRIU support will work only if the user gets OpenJ9 as part of a Semeru package (as opposed to building an SDK themselves), and a particular kind of package at that?

Reply:
@tajila - Please refer to Esther's comment regarding "CRIU support requires the version of CRIU that is a package in Semeru UBI container images." 

"Are we trying to say that the CRIU support will work only if the user gets OpenJ9 as part of a Semeru package (as opposed to building an SDK themselves), and a particular kind of package at that?"

Could you please clarify the point that she has raised? Thanks!

Reply:
@tajila - Will the CRIU support work only if the user gets OpenJ9 as part of a Semeru package? 

If this is an internal information and not something a user needs to know, then I will remove this line "CRIU support requires the version of CRIU that is a package in Semeru UBI container images."

Reply:
Also, shouldn't users use CRIU in non-container environments?

Reply:
> For example, are we trying to say that the CRIU support will work only if the user gets OpenJ9 as part of a Semeru package (as opposed to building an SDK themselves), and a particular kind of package at that?

We have package a custom version of CRIU that fixes known deficiencies with the library. Users can use the other versions of criu but we cant make any guarantees about the reliability of those versions. So we encourage users to use the version we package.

Reply:
> Also, shouldn't users use CRIU in non-container environments?

There are known issues when using criu in non-container environments. Its usable but it can be unreliable at times. So we only claim to support container environments.

Reply:
Done